AmplifyPlugins/Core/AWSPluginsCoreTests/Model/GraphQLRequest/GraphQLRequestAnyModelWithSyncTests.swift

```diff
@@ -10,11 +10,13 @@ import XCTest
 @testable import AmplifyTestCommon
 @testable import AWSPluginsCore
 
+// swiftlint:disable type_body_length
 class GraphQLRequestAnyModelWithSyncTests: XCTestCase {
 
     override func setUp() {
         ModelRegistry.register(modelType: Comment.self)
         ModelRegistry.register(modelType: Post.self)
+        ModelRegistry.register(modelType: CustomerOrder.self)
```

Comment:
add a new test file with CustomerOrder, `GraphQLRequestSyncCustomPrimaryTests`
- get, list, create, update, delete.

Reply:
keep in mind `get` query may pass but will fail AppSync service call, we would need to fix for successfully creating API integ test